packages/teleport/src/Discover/DownloadScript/DownloadScript.tsx

```diff
@@ -77,35 +78,55 @@ export function DownloadScript({
                 `}
               >
                 <Icons.Restore fontSize={4} />
-                {`Waiting for node   |   ${formatTime(
+                {`Waiting for Teleport SSH Service   |   ${formatTime(
```

Comment:
Are we re-rendering this entire component once a second to display the counter?

Reply:
yes, do you think it could be a problem? i think what we are rendering won't cause any performance problems because it's a pretty light component, but if you feel strongly about it, i can do refactoring on another PR since this PR is mostly about language/design tweaks.

Reply:
Nah I don't see it being an issue, I imagine we'll refactor it out at some point - I was just curious